spinoso-string/src/lib.rs

```diff
@@ -1684,6 +1705,89 @@ impl String {
         inner(self.inner.as_slice(), needle, offset)
     }
 
+    /// Returns the Integer byte-based index of the first occurrence of the
```

Comment:
Can we make this match the first line of the docs for the rust functions `index` and `rindex`. The talk of "Integer" I think comes straight from the Ruby docs but is a bit confusing in a Rust doc comment context.

Reply:
also fixed in d8eda7b7fe5eba53c460b4d80c28694da2905fe6